runtime/kusama/src/lib.rs

```diff
@@ -1733,15 +1733,15 @@ sp_api::impl_runtime_apis! {
 			Err(mmr::Error::PalletNotIncluded)
 		}
 
-		fn generate_batch_proof(_leaf_indices: Vec<u64>)
+		fn generate_batch_proof(_block_numbers: Vec<BlockNumber>)
 			-> Result<(Vec<mmr::EncodableOpaqueLeaf>, mmr::BatchProof<Hash>), mmr::Error>
 		{
 			Err(mmr::Error::PalletNotIncluded)
 		}
 
 		fn generate_historical_batch_proof(
-			_leaf_indices: Vec<u64>,
-			_leaves_count: u64,
+			_block_numbers: Vec<u64>,
+			_best_known_block_number: u64,
```

Comment:
Please use `BlockNumber` instead of `u64` (same for all runtimes)
```suggestion
			_block_numbers: Vec<BlockNumber>,
			_best_known_block_number: BlockNumber,
```